Formula/ktor.rb

```diff
@@ -0,0 +1,27 @@
+class Ktor < Formula
+  desc "CLI tool that creates porjects with ktor framework setup"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://github.com/ktorio/ktor-cli/archive/refs/tags/0.0.1-tmp.tar.gz"
+  sha256 "bdb311910a3afcebe3428aee6fd1ef8c19ac670d089f362a6fb89030c3edf873"
+  license "Apache-2.0"
+
+  depends_on "gradle" => :build
+  depends_on xcode: :build
+  depends_on arch: :x86_64
```

Comment:
Is there an issue upstream tracking arm support? If not, can you make one and reference it in a comment here?